mods/d2k/rules/ai.yaml

```diff
@@ -332,6 +250,9 @@ Player:
 			carryall: 4
 		SquadSize: 10
 		MaxBaseRadius: 40
+	AIHarvesterModule:
+	AIDamageReactionModule:
+	AISupportPowerModule:
 		SupportPowerDecisions:
```

Comment:
Nit: Could we rename this property? No need, IMO, to have `SupportPower` duplicated now that this lives under a support-power-specific trait.

Reply:
Will do that in #15786.